src/io/iohk/scalanet/test-resources/create-keystore.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash +x
```

Comment:
+1

Reply:
`/usr/bin/env bash` is usually the preferred approach. `/bin/bash` does not exist everywhere.